File: SPCode/CS/UI/Form/Support/Enums.cs
using System.Text.RegularExpressions;
using System.Diagnostics;
using System;
using System.Drawing.Drawing2D;
using System.Windows.Forms;
using System.Collections;
using System.Drawing;
using Microsoft.VisualBasic;
using System.Data.SqlClient;
using System.Data;
using System.Collections.Generic;
using WeifenLuo.WinFormsUI;
using Microsoft.Win32;
using WeifenLuo;

namespace SoftLogik.Win
{
	namespace UI
	{
		public enum SPRecordNavigateDirections
		{
			@None,
			First,
			Previous,
			@Next,
			Last
		}
		
		public enum SPFormRecordModes
		{
			InsertMode,
			EditMode,
			DirtyMode
		}
		
		public enum SPFormDataStates
		{
			@New,
			@Edited,
			@Deleted
		}
		
		public enum SPValidationItemCommands
		{
			@ValidateText,
			@ValidateNumber,
			@Validate,
			@ValidatePhone
		}
	}
	
}

File: SPCode/UI/Form/Support/Enums.vb
Namespace UI
    Public Enum SPRecordNavigateDirections
        [None]
        First
        Previous
        [Next]
        Last
    End Enum

    Public Enum SPFormRecordModes
        InsertMode
        EditMode
        DirtyMode
    End Enum

    Public Enum SPFormDataStates
        [New]
        [Edited]
        [Deleted]
    End Enum

    Public Enum SPValidationItemCommands
        [ValidateText]
        [ValidateNumber]
        [Validate]
        [ValidatePhone]
    End Enum
End Namespace
